Installing via go get fails

* **What are you trying to do?**

```shell
$ go get -u github.com/sonatype-nexus-community/nancy
# github.com/sonatype-nexus-community/nancy/audit
../go/src/github.com/sonatype-nexus-community/nancy/audit/auditlog.go:32:15: not enough arguments in call to aurora.Gray
        have (string)
        want (uint8, interface {})
# github.com/sonatype-nexus-community/nancy/ossindex
../go/src/github.com/sonatype-nexus-community/nancy/ossindex/ossindex.go:62:6: opts.Dir undefined (type func(string) badger.Options has no field or method Dir)
../go/src/github.com/sonatype-nexus-community/nancy/ossindex/ossindex.go:63:6: opts.ValueDir undefined (type func(string) badger.Options has no field or method ValueDir)
../go/src/github.com/sonatype-nexus-community/nancy/ossindex/ossindex.go:64:23: cannot use opts (type func(string) badger.Options) as type badger.Options in argument to badger.Open
../go/src/github.com/sonatype-nexus-community/nancy/ossindex/ossindex.go:161:16: txn.SetWithTTL undefined (type *badger.Txn has no field or method SetWithTTL)
```

fails on both Linux and macOS (using Go 1.13).

* **What feature or behavior is this required for?**

To install Nancy

* **How could we solve this issue? (Not knowing is okay!)**

Fix the respective dependencies.

* **Anything else?**

cc @bhamail / @DarthHater

--- a/Gopkg.lock
+++ b/Gopkg.lock
@@ -57,11 +57,11 @@
 
 [[projects]]
   branch = "master"
-  digest = "1:be40f095cd741773905744f16c1f7a21fd9226ccd0529f019deb7da6d667f71c"
+  digest = "1:9aaab932241543f719cd66e365a499956772e1f419a27aee2bb7bc3bf35af24a"
   name = "github.com/logrusorgru/aurora"
   packages = ["."]
   pruneopts = "UT"
-  revision = "a7b3b318ed4e1ae5b80602b08627267303c68572"
+  revision = "94edacc10f9bc12e9ac613b16125c6c68a8f7626"
 
 [[projects]]
   digest = "1:40e195917a951a8bf867cd05de2a46aaf1806c50cf92eebf4c16f78cd196f747"

--- a/audit/auditlog.go
+++ b/audit/auditlog.go
@@ -29,7 +29,7 @@ func logPackage(noColor bool, idx int, packageCount int, coordinate types.Coordi
 	} else {
 		fmt.Println("["+strconv.Itoa(idx)+"/"+strconv.Itoa(packageCount)+"]",
 			aurora.Bold(coordinate.Coordinates),
-			aurora.Gray("   No known vulnerabilities against package/version"))
+			aurora.Gray(20-1,"   No known vulnerabilities against package/version"))
 	}
 }
 

--- a/go.mod
+++ b/go.mod
@@ -7,7 +7,7 @@ require (
 	github.com/dgryski/go-farm v0.0.0-20180109070241-2de33835d102 // indirect
 	github.com/dustin/go-humanize v1.0.0 // indirect
 	github.com/golang/protobuf v1.2.0 // indirect
-	github.com/logrusorgru/aurora v0.0.0-20181002194514-a7b3b318ed4e
+	github.com/logrusorgru/aurora v0.0.0-20190803045625-94edacc10f9b
 	github.com/pkg/errors v0.8.0 // indirect
 	github.com/shopspring/decimal v0.0.0-20180709203117-cd690d0c9e24
 	github.com/stretchr/testify v1.3.0

